crates/opte-api/src/ip.rs

```diff
@@ -189,7 +189,7 @@ impl SubnetRouterPair {
         if prefix == 0 {
             0
         } else {
-            let round = u8::from(prefix % 8 != 0);
+            let round = u8::from(!prefix.is_multiple_of(8));
             (prefix / 8) + round
```

Comment:
This is what's causing CI to fail. I'm not going to treat this as a meaningful `API_VERSION` bump.